slack-base/src/main/java/com/hubspot/slack/client/models/commands/SlashCommandIF.java

```diff
@@ -0,0 +1,29 @@
+package com.hubspot.slack.client.models.commands;
+
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface SlashCommandIF {
```

Comment:
This looks like it should be an abstract class, so that it can implement the following interfaces:

* `HasChannel`
* `HasUser`

I think it's likely also worth adding a `HasTeam` to capture the "team id" field for other params that apply to teams.